examples/README.md

```diff
@@ -54,6 +54,13 @@ Management](https://github.com/triton-inference-server/server/blob/main/docs/arc
 which allows backends to behave in a stateless manner and leave the
 state management to Triton.
 
+Besides, Triton provides the
+[*BLS*](https://github.com/triton-inference-server/bls_backend) backend that
```

Comment:
Actually, I wonder if we need to put the BLS example in a separate GitHub repository. For example, in the below section, we have some backend examples that are placed [here](https://github.com/triton-inference-server/backend/tree/main/examples). We can put BLS example in the same directory and add the pointer here. CC @tanmayv25

Reply:
I'm fine with either way, but to me it looks like minimal and recommended backends are some preliminary examples for creating user's own backend and are covered in the [Tutorial](https://github.com/triton-inference-server/backend/tree/main/examples#tutorial) whereas repeat and stateful backends demonstrate some advanced usage and are not covered in the Tutorial. I think it would be better if @tanmayv25 can make the final decision here.

Reply:
Moving to "backend/examples/backends/" makes more sense now. thank you both for the suggestion. With the PR under bls_backend repo is under review, should I make this change after the PR is approved?

Reply:
I am currently reviewing the PR in `bls_backend` repository, I think you can keep the existing PR for now to collect the 2nd round of review comment, and have a separate PR in `backend` repo with the up-to-date code change and start a new round of review in the new PR.